CMakeLists.txt

```diff
@@ -8,6 +8,39 @@ set(VCPKG_OVERLAY_TRIPLETS "${CMAKE_CURRENT_LIST_DIR}/overlay_triplets")
 set(VCPKG_OVERLAY_PORTS "${CMAKE_CURRENT_LIST_DIR}/overlay_ports")
 set(CMAKE_TOOLCHAIN_FILE "${CMAKE_SOURCE_DIR}/vcpkg/scripts/buildsystems/vcpkg.cmake")
 
+# It seems that we didn't checkout submodules. Performing this.
+# See https://cliutils.gitlab.io/modern-cmake/chapters/projects/submodule.html
+if(NOT EXISTS "${CMAKE_TOOLCHAIN_FILE}")
+  execute_process(COMMAND git submodule sync --recursive 
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+    RESULT_VARIABLE GIT_SUBMODULE_RESULT
+  )
+  if(NOT GIT_SUBMODULE_RESULT EQUAL "0")
+    message(WARNING "git submodule sync --recursive  failed with ${GIT_SUBMODULE_RESULT}")
+  endif()
```

Comment:
For some reason, this command doesn't work on git for windows (git bash) distribution

Reply:
This comment was about `git submodules --init --recursive`, but I picked wrong line :/